deploy/containers/app.yml

```diff
@@ -0,0 +1,76 @@
+base_image: "rubybench/rubybench_docker"
+
+update_pups: false
+
+params:
+  db_name: rubybench_production
+  db_user: rubybench
+  home: /var/www/ruby-bench-web
+
+templates:
+  - "templates/redis.template.yml"
+  - "templates/postgres.template.yml"
+
+expose:
+  - "3000:3000"
```

Comment:
@SamSaffron I think I got it to work because this is exposing the port directly. It doesn't seem like it is going through `nginx`. Not sure if I'm supposed to be doing this.